subcommands/devices/cmd.go

```diff
@@ -59,11 +59,21 @@ func NewCommand() *cobra.Command {
 	updatesCmd.Flags().IntVarP(&listLimit, "limit", "n", 0, "Limit the number of updates displayed.")
 
 	cmd.AddCommand(configCmd)
-	cmd.AddCommand(updatesCmd)
+	addUuidCommand(cmd, updatesCmd)
+
 	return cmd
 }
 
-func getDeviceApi(_ *cobra.Command, name string) client.DeviceApi {
+func addUuidCommand(parent, command *cobra.Command) {
```

Comment:
I think the `addUuidFlag` name would look more intuitive.
We have functions with `addXyzFlag` already, so it is a de-facto pattern already.